flow/record/adapter/sqlite.py

```diff
@@ -58,12 +56,12 @@ def create_descriptor_table(con: sqlite3.Connection, descriptor: RecordDescripto
     column_defs = []
     for column_name, fieldset in descriptor.get_all_fields().items():
         column_type = FIELD_MAP.get(fieldset.typename, "TEXT")
-        column_defs.append(f"   `{column_name}` {column_type}")
+        column_defs.append(f'   "{column_name}" {column_type}')
     sql_columns = ",\n".join(column_defs)
 
     # Create the descriptor table
-    sql = f"CREATE TABLE IF NOT EXISTS `{table_name}` (\n{sql_columns}\n)"
-    logger.debug(sql)
+    sql = f'CREATE TABLE IF NOT EXISTS "{table_name}" (\n{sql_columns}\n)'
+    logging.debug(sql)
```

Comment:
Isn't it recommended to use a module specific logger?

Reply:
I changed it to global due to subclassing, but I think i'll just add a logger to the class.